data.sql

```diff
@@ -14,3 +14,25 @@ values
 ('Blossom','1998,10-13',3,17,true),
 ('Ditto','2022-05-14',4,22,true);
 
+insert into owners (full_name,age) values 
+('Sam Smith',34),
+('Jennifer Orwell',19),
+('Bob',45),
+('Melody Pond',77),
+('Dean Winchester',14),
+('Jodie Whittaker',38)
+;
+
+insert into species (name) values 
+('Pokemon'),
+('Digimon')
+;
+-- update animals table
+update animals set species_id='1' where name not like '%mon';
+update animals set species_id='2' where name like '%mon';
+
+update animals set owner_id=1  where name='Agumon';
```

Comment:
- [ _Optional_ ] Instead of hard coding the owner ID, you can have a nested query to get the owner you want. For example
```sql
  SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
  WHERE name = 'Agumon';
```